pandas/tests/frame/test_query_eval.py

```diff
@@ -202,6 +202,13 @@ def test_eval_simple(self, engine, parser):
         expected = df["a"]
         tm.assert_series_equal(expected, res)
 
+    def test_extension_array_eval():
```

Comment:
You have indented your test function as if it was a class method. That is likely why test are failing.